packages/oasgraph-cli/README.md

```diff
@@ -19,29 +19,49 @@ npm i -g oasgraph-cli
 
 
 ## Usage
-The basic usage of the CLI takes the specified OAS, create a GraphQL interface for it, and start a server to host the GraphQL interface.
 
-You can specify a local file containing the OAS specification or a remote url such as `http://127.0.0.1:3000/openapi.json`. Additionally, you can specify an optional port number so you can have multiple GraphQL servers for testing in the same machine.
+```
+Usage: oasgraph <OAS JSON file path or remote url> [options]
+
+Options:
+  -V, --version             output the version number
```

Comment:
I think this is a bit unspecific - both the documentation and the display of the version. Are we talking the version of OASGraph or the CLI here? I propose to output both, and make that clear in the docs also.

Reply:
The `-V` and `--version` options are auto-generated by Commander however I will see if I can overwrite them.

Reply:
Unfortunately, I don't think this can be changed easily

```
alans-mbp:oasgraph-cli alan.cha$ node lib/oasgraph.js -h
Usage: oasgraph <OAS JSON file path or remote url> [options]

Options:
  -V, --version             output the version number
  -V, --version             output the version number of the OASGraph library and CLI
  -p, --port <port>         select the port where the server will start
  -s, --strict              create a GraphQL schema matching the given OAS perfectly
  -a, --addSubOperations    nest operations based on path hierarchy
  -f, --fillEmptyResponses  wrap specific operations with no response schema
  --no-viewer               create artificial constructs that pass authentication credentials
  --save <file path>        save schema to path
  -h, --help                output usage information
```

It trying to change the version makes the overall code very ugly

Reply:
Ok, then let's stick with the original